tests/project/test_task1.py

```diff
@@ -19,5 +23,144 @@ def test_P1():
     pass
 
 
+def check_production(G: Graph, e: Node):
```

Comment:
The name of this function is kinda unclear to me.

Reply:
Thats on purpose